portfolio/src/main/webapp/script.js

```diff
@@ -56,4 +56,19 @@ function showSlides(n) {
     slides[slideIndex - 1].style.display = "block";
     dots[slideIndex - 1].className += " active";
     captionText.innerHTML = dots[slideIndex - 1].alt;
+}
+
+function showComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
```

Comment:
Nit: fix indentation.

Reply:
Indentation has been fixed in the commit 23f130a.